docs/concepts/json_schema.md

```diff
@@ -270,13 +461,75 @@ print(Foo.model_json_schema())
 """
 ```
 
-!!! note
-    Defaults can be set outside `Annotated` as the assigned value or with `Field.default_factory` inside `Annotated`.
-    The `Field.default` argument is not supported inside `Annotated`.
+### Model-Level Customization
+
+You can also use [model config][pydantic.config.ConfigDict] to customize JSON schema generation on a model.
+Specifically, the following config options are relevant:
+
+* [`title`][pydantic.config.ConfigDict.title]
+* [`json_schema_extra`][pydantic.config.ConfigDict.json_schema_extra]
+* [`schema_generator`][pydantic.config.ConfigDict.schema_generator]
+* [`json_schema_mode_override`][pydantic.config.ConfigDict.json_schema_mode_override]
+
+### Using `json_schema_extra`
+
+The `json_schema_extra` option can be used to add extra information to the JSON schema, either at the
+[Field level](#field-level-customization) or at the [Model level](#model-level-customization).
+You can pass a `dict` or a `Callable` to `json_schema_extra`.
+
+#### Using `json_schema_extra` with a `dict`
+
+You can pass a `dict` to `json_schema_extra` to add extra information to the JSON schema:
 
-For versions of Python prior to 3.9, `typing_extensions.Annotated` can be used.
+```py
+from pydantic import BaseModel, ConfigDict
+
+
+class Model(BaseModel):
+    a: str
+
+    model_config = ConfigDict(json_schema_extra={'examples': [{'a': 'Foo'}]})
+
+
+print(Model.model_json_schema())
+"""
+{
+    'examples': [{'a': 'Foo'}],
+    'properties': {'a': {'title': 'A', 'type': 'string'}},
+    'required': ['a'],
+    'title': 'Model',
+    'type': 'object',
+}
+"""
+```
+
+#### Using `json_schema_extra` with a `Callable`
+
+You can pass a `Callable` to `json_schema_extra` to modify the JSON schema with a function:
+
+```py
```

Comment:
same

Reply:
Changing all of these, but also adding the `json.dumps(..., indent=2)` for consistency.